### DISCUSSION TITLE
Auto-Add Studio Or Network Shows

### DISCUSSION THREAD

**Discussion by Festeazy** [sct/overseerr#2396] [Feature Requests]
I would like to see the ability to set up a method to automatically add shows from a network or studio, with the following filters:

options to set All or Upcoming 
Option between shows or movies 
checkboxes for ratings

**Comment by TheCatLady** [comment]
IMO this isn't something that would be handled by Overseerr, which is a request platform.  You can use the import lists functionality in Radarr/Sonarr to accomplish this.
